plugin/interface_generator/main.go

```diff
@@ -265,6 +265,7 @@ func getPluginInfo(dir string) (*PluginInterfaceInfo, error) {
 	packages, err := parser.ParseDir(pluginInfo.FileSet, dir, nil, parser.ParseComments)
 	if err != nil {
 		log.Println("Parser error in dir "+dir+": ", err)
+		return nil, err
```

Comment:
@crspeller - Any thoughts on this?

Reply:
Looks like the return was just forgotten?

Reply:
Yeah, the docs say that on an error, a partial map is returned with whatever was parsed. So just checked whether it was intentional or not. Thanks for the clarification!